adhocracy4/filters/filters.py

```diff
@@ -2,7 +2,22 @@
 from django.http import QueryDict
 
 
-class DefaultsFilterSet(django_filters.FilterSet):
+class PagedFilterSet(django_filters.FilterSet):
+    """Removes page parameters from the query when applying filters."""
+    page_kwarg = 'page'
+
+    def __init__(self, data, *args, **kwargs):
+        if self.page_kwarg in data:
+            try:
+                data.pop(self.page_kwarg)
+            except AttributeError:
+                # Create a mutable copy
+                data = data.copy()
+                data.pop(self.page_kwarg)
```

Comment:
I think the query dict will always be immutable by default (so I would only implement that case) and there is also `QueryDict.mutable`, which is probably more than catching an attribute error.